microservices/s3_to_redshift/s3_to_redshift.py

```diff
@@ -139,9 +138,24 @@ def copy_query(dbtable, batchfile, log):
 
         obj = client.get_object(Bucket=bucket, Key=object_summary.key)
         body = obj['Body']
+        csv_string = ''
+        if 'global_regex' in data:
+            body_stringified = body.read()
+            for line in body_stringified.splitlines():
+                for exp in data['global_regex']['regex']:
+                    parsed_line, matched = re.subn(data['global_regex']['regex'][exp]['match'],
+                        data['global_regex']['regex'][exp]['replace'],
+                        line.replace(data['global_regex']['string_replace']['match'],
+                            data['global_regex']['string_replace']['replace']))
+                    if matched:
+                        parsed_line = parsed_line + "\r\n"
```

Comment:
why does this include `"\r\n"` when you have `"linefeed": "\r\n"` in your configuration file?

Reply:
Ive changed the code to use the linefeed from the config, if defined. esle, it defaults to \r\n.